packages/site-client/src/entrypoints/docs.ts

```diff
@@ -0,0 +1,12 @@
+/**
+ * @license
+ * Copyright 2023 Google LLC
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import '../components/wco-docs-page.js';
+import type {NavEntry} from '../components/wco-docs-page.js';
+import {html} from 'lit';
+
+export const renderDocsPage = (navEntries: NavEntry[]) =>
+  html`<wco-docs-page .navEntries=${navEntries}></wco-docs-page>`;
```

Comment:
oops... we're missing the content here. This needs to be something like:

```suggestion
export const renderDocsPage = (content: unknown, navEntries: NavEntry[]) =>
  html`<wco-docs-page .navEntries=${navEntries}>${content}</wco-docs-page>`;
```

You can see the problem on the site at /docs/ where the content is missing.

Reply:
Fixed